### PR TITLE
Deploy scroll to testnet

### PR DESCRIPTION
**Pull request by bruce-riley**

### CODE REVIEW

**Review by panoel — DISMISSED** [review]
I only looked at the node/* files

**Review by evan-gray — COMMENTED** [review]
If Scroll supports `finalized` and `safe` can we add it to the list in `getFinality`?

**Review by evan-gray — APPROVED** [review]

**Review by panoel — APPROVED** [review]

**Review by kcsongor — APPROVED** [review]

**Review by a5-pickle — APPROVED** [review]
rust/core looks good

**Review by kev1n-peters — APPROVED** [review]

File: devnet/node.yaml [hunk 1]
```diff
@@ -103,6 +103,8 @@ spec:
             - ws://eth-devnet:8545
             - --baseRPC
             - ws://eth-devnet:8545
+            - --scrollRPC
+            - ws://eth-devnet:8545
```
Comment (SEJeff) [inline devnet/node.yaml:107]: Won't this fail the guardian start due to [this code](https://github.com/wormhole-foundation/wormhole/pull/3429/files#diff-7f43e1a5bbb6083b5e470710e19e804d778f4a411b23fdbef1b2955f56b67bbdR637-R639) added in this PR? Seems like a contract needs to be specified and I'm clearly missing some magic here.

**Review by SEJeff — APPROVED** [review]

File: devnet/node.yaml (hunk 1 above)
Comment (bruce-riley) [reply devnet/node.yaml:107]: No, this is not a problem. In tilt (unsafeDevMode) the contract address for all the EVM chains are set to the one deployed locally. See line node.go, line 498.

**Pull request merged**